litex/tools/litex_json2dts_linux.py

```diff
@@ -44,18 +44,35 @@ def generate_dts(d, initrd_start=None, initrd_size=None, polling=False):
     if initrd_size is None:
         initrd_size = default_initrd_size
 
+    if initrd == "enabled" or initrd is None:
+        initrd_enabled = True
+    elif initrd == "disabled":
```

Comment:
Not a great fan of using `enabled` and `disabled` as special values.

I guess it won't be a very common scenario but still those **could** be a valid initrd file names.

Reply:
That right, but this suggested, it seems pretty reasonable.

Reply:
Sure, I don't have a nice solution to this myself. Having said that I don't think this is a blocker.